lua/wire/stools/expression2.lua

```diff
@@ -215,6 +215,7 @@ if SERVER then
 	end)
 
 	function TOOL:Upload(ent)
+		ent.AwaitingUpload = true
```

Comment:
I think this ought to go in the `WireLib.Expression2Upload`

Reply:
Good point, done